Auth.tsx

```diff
@@ -50,10 +53,38 @@ const AuthScreen = () => {
     }
   };
 
+  const handleWhoami = async () => {
+    if (!decryptedData) {
+      console.error("unable to get whoami; must have decrypted data");
+      return;
+    }
+
+    const publicKey = uint8ArrayToHexString(
+      getPublicKey(uint8ArrayFromHexString(decryptedData), true)
+    );
```

Comment:
so in other words: take the private key data (hex string), then parse the bytes out, then get the public key for the private key bytes, then encode as a hex string.

I think that's worth a small comment or a small function extraction? Not a huge deal obviously, I did figure out what this was doing eventually :)